### DISCUSSION TITLE
Hey MaxiHuHe04!

### DISCUSSION THREAD

**Discussion by Casper-vdVeen** [MaxiHuHe04/iTunes-Backup-Explorer#29] [General]
Hi Maxi, 

My name is Casper, I'm currently helping a friend recover her iphone files, which she has backed up on an Acer laptop. The problem I'm facing however is that whenever I want to select the backed up files in your program, they all appear empty. I figured that the files aren't linked to a device.

My question is, what's the best way to reach you about this? I'd love to hear from you, if you have the time.

Kind regards,
Casper vd Veen

**Comment by MaxiHuHe04** [comment]
Hey, right here is probably the easiest. If you don't want it to be public, you could also email me at maxiherczegh [at] gmail.com or add me on Discord (maxihuhe04).

Which files exactly appear empty?

**Comment by fungibility42** [comment]
I may have a similar issue. When I navigate to the backup folder (to open a backup), it won't allow me to open anything other than a few .db or .plist files. But when I open those, there doesn't seem to be any data shown in the program (by search or export). It seems like it's not reading the backup itself (which is saved into a bunch of numbered/lettered folders with filenames like 00008140-000E65422193001C/0a). Is there an issue with getting the program to recognize the backup?

**Reply by MaxiHuHe04** [reply]
Whether you open .db or .plist files doesn't matter. The program always reads multiple files from the backup folder where the database and manifest files are in.
Is the Files tab also empty or just the File search tab? In the file search, you have to type in something in the search boxes (e.g. `%` for "anything") and click on Search for files to start appearing.

**Reply by MaxiHuHe04** [reply]
Does the Information tab show plausible information about your backup?
